study.md

```diff
@@ -57,7 +57,7 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+CREATE. No it does not persist objects as rows in the database, it requires another method.
```

Comment:
`#create` will instantiate a new object and save it to the database.

`#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.